lib/Conversion/TritonGPUToLLVM/TritonGPUToLLVMBase.h

```diff
@@ -421,6 +421,47 @@ class ConvertTritonGPUOpToLLVMPatternBase {
   // -----------------------------------------------------------------------
   // Utilities
   // -----------------------------------------------------------------------
+  Value getMask(Type valueTy, ConversionPatternRewriter &rewriter,
```

Comment:
Could this function also be used for `atomicCAS` and `atomicRMW`?

Reply:
Yep, I just wanted to test that separately

Reply:
I thought maybe for LoadOp as well, currently we don't mask

Reply:
> I thought maybe for LoadOp as well, currently we don't mask

Why should we apply it for load?

Reply:
If the cache line gets evicted between two threads loading replicated data, that could be a perf hit

Reply:
Hmm, let me think about that some more